app/controllers/defendants_controller.rb

```diff
@@ -1,33 +1,27 @@
 # frozen_string_literal: true
 
 class DefendantsController < ApplicationController
-  include DefendantSearchable
-
-  before_action :load_and_authorize_search
+  before_action :authorize_defendant_search
   before_action :set_unlink_reasons,
                 :set_unlink_attempt,
                 :set_defendant_if_required
 
   add_breadcrumb :search_filter_breadcrumb_name, :new_search_filter_path
   add_breadcrumb :search_breadcrumb_name, :search_breadcrumb_path
 
-  add_breadcrumb (proc { |v| v.prosecution_case_name(v.controller.defendant.prosecution_case_reference) }),
-                 (proc { |v| v.prosecution_case_path(v.controller.defendant.prosecution_case_reference) })
+  # TODO: I have (temporarily) removed the case breadcrumb as this was
```

Comment:
try changing any link_to that use this to pass the urn if available?! it worked on my branch with a hearing controller

Reply:
👍 done - many thanks for your assistance with getting the case breadcrumb working